Header header
float32[4] foot_forces
float32[3] imu_euler
float32[3] base_velocity
string current_gait_mode
bool is_safe

